frontend/src/components/smart/CharacterList.vue

```diff
@@ -2,41 +2,41 @@
   <div>
     <div class="filters row mt-2 pl-2" v-if="showFilters" @change="saveFilters()">
       <div class="col-sm-6 col-md-6 col-lg-2 mb-3">
-        <strong>Level</strong>
+        <strong>{{$t('characterList.level')}}</strong>
         <select class="form-control" v-model="levelFilter">
           <option v-for="x in ['', 1, 11, 21, 31, 41, 51, 61, 71, 81, 91]" :value="x" :key="x">
-            {{ x ? `${x} - ${x + 9}` : 'Any' }}
+            {{ x ? `${x} - ${x + 9}` : $t('characterList.sorts.any') }}
           </option>
         </select>
       </div>
 
       <div class="col-sm-6 col-md-6 col-lg-2 mb-3">
-        <strong>Element</strong>
+        <strong>{{$t('characterList.element')}}</strong>
         <select class="form-control" v-model="elementFilter">
-          <option v-for="x in ['', 'Earth', 'Fire', 'Lightning', 'Water']" :value="x" :key="x">{{ x || 'Any' }}</option>
+          <option v-for="x in ['', 'Earth', 'Fire', 'Lightning', 'Water']" :value="x" :key="x">{{ x || $t('characterList.sorts.any') }}</option>
```

Comment:
Missing transaltion for elements in options

Reply:
Fixed in d565096.
Translation strings added to the v-for. 
The english strings are still used as a value for the filter. Using the translated strings as the value invokes other issues. E.g. session storage for the filters would need to be cleared after changing the language and icons for the elements wouldn't be shown, because the icon classes are using the english strings as well.